src/util/log.d

```diff
@@ -339,8 +351,8 @@ class FileLogger(alias Layout) : Logger
             buffer.swap(this.buffer);
         }
 
-        Layout(buffer, eventInfo, putMessage);
-        file.lockingTextWriter.put(buffer.data);
+        layout.write(appender, eventInfo, putMessage);
```

Comment:
What `appender`?
`buffer` is prepared to do the job...

Reply:
fixed